rudder-agent/SOURCES/debug-script/tests/rudder_packages_agent

```diff
@@ -65,6 +65,27 @@ rudder_packages_agent() {
 
   fi
 
+  # We check if ".rpmnew" packages exist in the directories that keep the configuration files.
+  PATH_DIRECTORY1="/opt/rudder/etc/"
+  PATH_DIRECTOTY2="/etc/httpd/"
+
+  if [ -d "${PATH_DIRECTORY1}" -a -d "${PATH_DIRECTOTY2}" ]
+  then
+    TEST1=$( find "${PATH_DIRECTORY1}" -type f \( -name "*.rpmnew" \) | wc -l )
+    TEST2=$( find "${PATH_DIRECTOTY2}" -type f \( -name "*.rpmnew" \) | wc -l )
+    RESULT=$(($TEST1+$TEST2))
+
+    if [ "${RESULT}" -ne 0 ]
+    then
+      test_failure "Package version is incoherent" "${RESULT} packages found to update"
```

Comment:
The message is not correct.
It is more like "there are unapplied configuration files (.rpmnew)"